src/vivarium/component.py

```diff
@@ -159,54 +158,47 @@ def name(self) -> str:
 
     @property
     def sub_components(self) -> List["Component"]:
-        """
-        Provide components managed by this component.
+        """Provide components managed by this component.
 
         Returns
         -------
         List[Component]
```

Comment:
Wasn't this supposed to be removed?

Reply:
This (and the `str` return type above it) were the two I reffered to earlier that for some reason sphinx has a problem with removing.